src/link/Coff.zig

```diff
@@ -388,6 +388,7 @@ fn populateMissingMetadata(self: *Coff) !void {
         self.rdata_section_index = try self.allocateSection(".rdata", file_size, .{
             .CNT_INITIALIZED_DATA = 1,
             .MEM_READ = 1,
+            .MEM_WRITE = 1,
```

Comment:
rdata is supposed to be read-only isn't it?

Reply:
Due to a linker bug mutable globals are ending up here.  I don't understand this enough to open and issue, but @kubkon said to just do this for now.

Reply:
It's not rdata, it's `.data.rel.ro` since we may have pointers inside a struct we lower in that section. The linker code path responsible for selecting output section doesn't do a recursive check for embedded pointers in any linker (macho, elf or coff), and this is important for PIC/PIE targets where the dynamic linker has to slide the address of each pointer within the binary - hence `.data.rel.ro` which is only writeable during dynamic relocation startup phase - well, in theory at least.

Reply:
Anyhow, as @jacobly0 mentioned, that was my suggestion to make it writeable to unblock his progress. I am aware of this nuance and will provide a fix in ELF first, and then backport it to other linkers as I start working on them.